srcpkgs/ocs-url/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'ocs-url'
+pkgname=ocs-url
+version=3.1.0
+revision=7
+wrksrc="ocs-url-release-3.1.0"
+hostmakedepends="qt5-qmake qt5-svg-devel qt5-svg kdeclarative kdeclarative-devel"
+depends="qt5"
+short_desc="Install helper program for OpenCollaborationServices (ocs://)"
+maintainer="ThatGeekyWeeb <https://github.com/ThatGeekyWeeb/> <thatgeekyweeb@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://www.opencode.net/dfn2/ocs-url"
+distfiles="https://www.opencode.net/dfn2/ocs-url/-/archive/release-${version}/ocs-url-release-${version}.tar.gz https://github.com/akiraohgaki/qtil/archive/master.tar.gz"
+checksum="d8c67920359c36eb0004221e5b5695fd04cbc9e93811c372837bdb2a8c9cbcc8 cabe912cf06568a32925602f6f5374e3be22a02b705466adcff8573e181e83f1"
+nostrip="true"
+patch_args="--forward --strip=1"
+do_install() {
+	mkdir ${wrksrc}/lib/qtil
+	mv ${wrksrc}/../qtil-master/* ${wrksrc}/lib/qtil/
+	cd ${wrksrc}
+	qmake PREFIX="/usr"
+	make -j${nproc}
+	make INSTALL_ROOT=${DESTDIR} install -j${nproc}
```

Comment:
You shouldn't run these steps manually. Please try to use the `qmake` build style.

Reply:
use the qmake build style

Reply:
Ok, sure, although, it would be great if you could inform me of what that is, as my knowledge of the qmake build style, is, non-existent

Reply:
https://github.com/void-linux/void-packages/blob/master/Manual.md#build-style-scripts

Reply:
Alright, understood! resolved
```sh
build_style="qmake"
configure_args="PREFIX='/usr'"
make_build_args="INSTALL_ROOT=${DESTDIR} -j${npoc}"
make_install_args="INSTALL_ROOT=${DESTDIR} -j${npoc}"
```